.rubocop.yml

```diff
@@ -17,18 +17,19 @@ Lint/ImplicitStringConcatenation:
     - 'lib/generators/sufia/**/*'
 
 Metrics/LineLength:
-  Max: 400
+  Enabled: false
 
 Metrics/AbcSize:
-  Max: 30
-  Exclude:
-    - 'app/controllers/concerns/sufia/admin/depositor_stats.rb'
-    - 'app/controllers/concerns/sufia/my_controller_behavior.rb'
-    - 'app/services/sufia/user_stat_importer.rb'
-    - 'lib/sufia/arkivo/metadata_munger.rb'
+  Enabled: false
 
 Metrics/MethodLength:
-  Max: 15
+  Enabled: false
```

Comment:
Any particular reason we don't want to pick a reasonable limit to try to live within for this? it can always be disabled on a case-by-case basis